docs/02-get-started/05-observability.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Observability
+---
+
+We successfully got our Function [triggered](04-trigger-workload-with-event.md).
+Now we would like to check its logs. 
+To do that, we'll use the Grafana dashboard that comes with Kyma. 
+
+> **NOTE:** See how to access logs from the Function's Pod [via Kyma Dashboard](../04-operation-guides/operations/obsv-01-access-logs.md#kubernetes-logs-in-kyma-dashboard) and [using kubectl](../04-operation-guides/operations/obsv-01-access-logs.md#kubernetes-logs-using-kubectl). 
+
+1. To access Grafana, forward a local port to a port on the service's Pod:
+    ```bash
+    kubectl -n kyma-system port-forward svc/monitoring-grafana 3000:80
+    ```
+2. In your browser, go to [`http://localhost:3000`](http://localhost:3000) to open Grafana dashboard.
+3. Using the left menu, navigate to **Explore** and choose **Loki** from the dropdown list.
+4. Click on **Log browser** and select the following values:
+   - **1. Select labels to search in**: `container`, `function`
+   - **2. Find values for the selected labels**: for **function** choose `lastorder`, for **container** choose `function` 
+   
+   Click **Show logs**.
```

Comment:
isn't this simply a 5th step?
```suggestion
5. Click **Show logs**.
```

Reply:
It's here on purpose as part of this steps for two reasons: 
1) the button is in the area where you fill in this stuff, it's like one block, so it helps the user focus visually and find the button (Grafana is visually a bit chaotic to me TBH)
![image](https://user-images.githubusercontent.com/53601578/133629195-ef5e882b-3ea1-4f90-a349-6cc85a058425.png)

2) The alternative that's right below this is an alternative for steps 2-3, so this way it's clear and I don't need to add another sentence. 

I'd like to leave it this way :)